lib/bundle.js

```diff
@@ -164,7 +165,7 @@ exports.bundle = function(moduleExpression, fileName, opts) {
     if (opts.injectConfig)
       ui.log('ok', '`' + output.bundleName + '` added to config bundles.');
 
-    logBuild(path.relative(process.cwd(), fileName), opts);
+    logBuild(path.relative(env.cwd(), fileName), opts);
```

Comment:
Are you sure this is necessary? This should still work with `process.cwd()` I think?